lib/auth/permissions.go

```diff
@@ -358,7 +360,8 @@ func GetCheckerForBuiltinRole(clusterName string, clusterConfig services.Cluster
 			role.String(),
 			services.RoleSpecV3{
 				Allow: services.RoleConditions{
-					Namespaces: []string{services.Wildcard},
+					Namespaces:     []string{services.Wildcard},
+					DatabaseLabels: services.Labels{services.Wildcard: []string{services.Wildcard}},
```

Comment:
Same here. DB should be able to read their own record instead of this clause. See auth_with_roles `GetUser` for an example of an exception clause.

Reply:
Makes sense, I can work on that.